.github/workflows/tests_linux_x86.yml

```
@@ -0,0 +1,168 @@
+name: Tests::Linux::x86_64
+on:
+  release:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  COVERAGE_FLAGS: "--cov=pennylane_lightning_gpu --cov-report=term-missing --cov-report=xml:./coverage.xml --no-flaky-report -p no:warnings --tb=native" 
+  GCC_VERSION: 11
+  OMP_NUM_THREADS: "2"
+  CI_CUDA_ARCH: 86
+
+concurrency:
+  group: gpu-test-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  cpptests:
+    runs-on:
+      - self-hosted
+      - ubuntu-22.04
+      - gpu
+
+    strategy:
+      max-parallel: 1
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.8'
+
+      - name: Remove Ubuntu unattended upgrades
+        run: sudo apt-get remove unattended-upgrades
+
+      - name: Install required packages
+        run: |
+          sudo apt-get update && sudo apt-get -y -q install gcc-${{ env.GCC_VERSION }} g++-${{ env.GCC_VERSION }} gcovr lcov
```

I was wondering, could we try, the following? (This is instead of removing unattended-upgrades

```suggestion
          sudo apt-get -o DPkg::Lock::Timeout=60 -y -q install gcc-${{ env.GCC_VERSION }} g++-${{ env.GCC_VERSION }} gcovr lcov
```

Rationale for this change:
- Unattended upgrades ensures system packages are updated similar to apt-get update
- The DPkg flag being passed ensures this lines waits for unattended upgrades to finish (for up to 60 seconds) before timing out. This can be set to -1 for indefinite.

---

Yep yep non issue! As long it consistently works